docs/keepassxc-browser-manual.md

```diff
@@ -0,0 +1,333 @@
+Only after this KeePassXC-Browser is able to read the updated settings.
+
+### 1.3. Detection of credential fields
+
+After a page is loaded KeePassXC-Browser starts to search all **visible** input fields.
+
+The detection works for the following situations:
+- A username field is followed with a password field. These are handled as a combination.
+- Only a password field is visible. A username field was present in the previous page.
+- Only username field is visible. This needs a whitelisting. See [2.4. Settings: Site preferences](#24-settings-site-preferences).
+
+The auto detection of input fields are called every time the page is refreshed or changed dynamically and there are visible input fields present.
+Any input field that is hidden, outside the page view or too small to be visible are ignored.
+
+When username and/or password fields are not detected, you can click on the KeePassXC-Browser icon and press button *Redetect credential fields* from the popup.
+
+### 1.4. Choose custom credential fields for a page
+
+Sometimes a page can contain other input fields between the username and password field. In these cases KeePassXC-Browser might not detect
+the fields correctly.
+
+It is possible to define the fields manually. Click the KeePassXC-Browser icon and select *Choose custom credentials fields for this page*:
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser choose custom credential fields" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_choose_custom_credential_fields.png">
+</div>
+
+At first all input fields are highlighted. Click on the field you want to use as a username field or skip this step if no username field is required.
+Next, choose a password field and in the last step confirm your selection.
+
+<div style="text-align: center;">
+<img alt="KeePassXC-Browser custom credential fields" style="max-width: 100%;"
+src="{{ site.baseurl }}/images/browser-migration/manual_custom_credential_fields.png">
+</div>
+
+Additional fields called *String Fields* can be selected before the confirm. This functionality is explained in [1.7. Fill additional values using String Fields](#17-fill-additional-values-using-string-fields).
+
+Next time you open the page KeePassXC-Browser will use the defined input fields and no longer auto detect them. The custom credential fields can be
+modified or removed from *Settings -> Custom credential fields*, or by reopening the selection from the popup and selecting *Discard selection*.
+
+### 1.5. Create new or modify existing credentials
+
+KeePassXC-Browser is able to detect form submits. If unsaved or modified credentials are detectd, the KeePassXC-Browser icon starts blinking red.
+When clicked, it remains red until you add, update or dismiss the detected changed. It is also possible to put the site to an ignore list when no 
+credential changes are informed in the future.
```

Comment:
"when no credential changes are informed in the future" → "so that no credential changes are detected in the future"